code/modules/vending/megaseed.dm

```diff
@@ -52,8 +52,8 @@
 					  /obj/item/seeds/random = 2)
 	premium = list(/obj/item/reagent_containers/spray/waterflower = 1)
 	refill_canister = /obj/item/vending_refill/hydroseeds
-	default_price = 10
-	extra_price = 50
+	default_price = 100
+	extra_price = 350
```

Comment:
How does this affect a botanist's ability to buy strange seeds?

Reply:
Ah, this was already brought up. Still, I'd like to know how easy/hard it will be to scrape up the funds for strange seeds after this PR goes through. How many will you be able to buy with your roundstart cash?

Reply:
Oh, this was already brought up. Still, I'd like to know how hard it'll be to scrape up the funds needed to buy strange seeds as a botanist after this PR gets merged. How many strange seeds will I be able to buy with my roundstart credit supply?

Reply:
I tested this for two seconds locally because I was curious just how severe the effect would be. I latejoined like 10 minutes in (not sure if that changes anything, I haven't paid enough attention to how this works) as Botanist with $750. All the contraband in the MegaSeed Servitor cost $350 each. Including that dumb water flower. Payday was $125. Diethylamine and Ammonia in the NutriMax were $250.

Reply:
either stranges are a luxury item or they're not. 

I'm tired of the fragile balancing act of "they need to have a cost but never enough to actually matter for... reasons". Either it's integral to the occupation in which should be free, it's a luxury item that should come at a cost that's noticeable for the botanist, or it should be not obtainable through this means.

what they have now is fine atm, especially since this was more putting it in line with other costs.

Reply:
I don't bother with them and would agree they're 'luxury' (we're just doing our jobs playing this after all) items, especially if there's only two of them.

Diethylamine requires like 100u to increase yield though, so waiting 10 minutes to afford two bottles is dumb imo. I know you (cob) hate botany and there are less lazy ways to increase yield but I get like 110ms ping (which doesn't seem bad until you play on a nopop server hosted in the country next door and realise how fast you could be) so I find it painstaking enough already. I don't know how in the minority I am with this problem though. Maybe >80% are sugarblooded Americans with 5ms to server

Reply:
I don't hate botany, just the side of it that assumes it's meant to be a chemistry supplement as it's primary goal vs. a job where you chill and grow plants.

I have a hard time accepting the balancing metric of "things need to be easier to make it more friendly to people who have bad latency to our current servers". Time for an ~~European~~ Asian server :^)?

Like I said though i'm fine with this as is.